README.md

```diff
@@ -287,20 +299,125 @@ models
+above. However, it is important to see `libpython3.6m.so.1.0` in the list of
+linked shared libraries. If you use a different Python version, you should see
+that version instead. You need to copy the `triton_python_backend_stub` to the
+model directory of the models that want to use the custom Python backend
+stub. For example, if you have `model_a` in your [model repository](https://github.com/triton-inference-server/server/blob/main/docs/model_repository.md), the folder
+structure should look like below:
+
+```
+models
+|-- model_a
+    |-- 1
+    |   |-- model.py
+    |-- config.pbtxt
+    `-- triton_python_backend_stub
+```
+
+Note the location of `triton_python_backend_stub` in the directory structure above.
+
+### 2. Packaging the Conda Environment
+
+It is also required to create a tar file that contains your conda environment.
+Currently, Python backend only supports
+[conda-pack](https://conda.github.io/conda-pack/) for this purpose.
+[conda-pack](https://conda.github.io/conda-pack/) ensures that your conda
+environment is portable. You can create a tar file for your conda environment
+using `conda-pack` command:
+
+```
+$ conda-pack
+Collecting packages...
+Packing environment at '/home/iman/miniconda3/envs/python-3-6' to 'python-3-6.tar.gz'
+[########################################] | 100% Completed |  4.5s
+```
+
+After creating the tar file from the conda environment, you need to tell Python
+backend to use that environment for your model. You can do this by adding the
+lines below to the `config.pbtxt` file:
+
+```
+name: "model_a"
+backend: "python"
+
+...
+
+parameters: {
+  key: "EXECUTION_ENV_PATH",
+  value: {string_value: "/home/iman/miniconda3/envs/python-3-6/python3.6.tar.gz"}
+}
+```
+
+### Important Notes
+
+1. The version of the Python interpreter in the tar file must match the version of triton_python_backend_stub.
+
+2. If you don't want to use a different Python interpreter, you can skip
+[Building Custom Python Backend Stub Step](#1-building-custom-python-backend-stub).
+In this case you only need to pack your environment using `conda-pack` and
+provide the path to tar file in the model config. However, the previous note
+still applies here and the version of the Python interpreter inside the conda
+environment must match the Python version of stub used by Python backend. The
+default version of the stub is Python 3.8.
+
+3. Note that if the Python version of your environment matches the Python
+version available inside the container, the packages installed in
+[conda](https://conda.io) or [virtualenv](https://virtualenv.pypa.io/) will be
+picked up by Python backend and applied to all the Python models. In this case
+you can skip both [Building Custom Python Backend
+Stub](#1-building-custom-python-backend-stub) and [Packaging the Conda
+Environment](#2-packaging-the-conda-environment) steps. You only need to make
+sure that the environment is activated before you start the Triton Server.
 
```

Comment:
Add a note describing that the same env can be used for multiple models by having the models point to the same tar file.